exchanges/poloniex/poloniex_wrapper.go

```diff
@@ -148,14 +148,14 @@ func (p *Poloniex) GetExchangeFundTransferHistory() ([]exchange.FundHistory, err
 }
 
 // GetExchangeHistory returns historic trade data since exchange opening.
-func (p *Poloniex) GetExchangeHistory(cP pair.CurrencyPair, assetType string) ([]exchange.TradeHistory, error) {
+func (p *Poloniex) GetExchangeHistory(pair pair.CurrencyPair, assetType string) ([]exchange.TradeHistory, error) {
```

Comment:
pair points to a package in currency folder, and sometimes it will give annoying errors in editors. Maybe change to cPair or declare package name to circumvent this.